src/main/java/busTickets/busTicketStorage/BusTicketStorage.java

```diff
@@ -0,0 +1,67 @@
+package busTickets.busTicketStorage;
+
+import busTickets.BusTicket;
+import busTickets.BusTicketValidation;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BusTicketStorage {
+    public ArrayList<BusTicket> busTickets = new ArrayList<>();
+
+    public BusTicketStorage() {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            objectMapper.findAndRegisterModules();
+
+            List<BusTicket> loadedTickets = objectMapper.readValue(
+                    new File("src//main//java//busTickets//busTicketStorage//busTickets.json"), new TypeReference<List<BusTicket>>() {}
+            );
+
+            busTickets.addAll(loadedTickets);
+        } catch (IOException e) {
+            System.err.println("Failed to load tickets from JSON file: " + e.getMessage());
+        }
+
+        for (BusTicket busTicket : busTickets) {
+            validateBusTicket(busTicket);
+        }
+    }
+
+    public static void validateBusTicket(BusTicket busTicket) {
```

Comment:
Why this method is static?

Reply:
I resolved it by moving this method entirely to class ValidationStatistics